ink/yuki/punishment_edging

```diff
@@ -261,13 +259,22 @@
 +[pose:puppy_yes]
   /perform react_laugh amused 60
   ~ addObey()
-    You're not supposed to enjoy it thaaat much!
+    Goodie! But remember, no cumming.
 
 +[pose:puppy_No]
+TODO set perform
   /perform
   ~ addDisobey()
-    -> Yuki_AcknowledgeNo
+    -> Yuki_AcknowledgeNo -> 
+    
+    //change to a shuffle and add lines
     Well maybe next time you shouldn't be such a naughty little puppy, hmmm?
+    
+    Maybe you should move those paws a little better then? 
+    
+    Good, you're being punished after all. 
+    
+    -> edging_break_1
```

Comment:
Yes, my intention is to add a shuffle specifically after the player says 'no' here, and the shuffle will currently contain the three possible lines: 
"Well maybe next time you shouldn't be such a naughty little puppy, hmmm?", 
"Maybe you should move those paws a little better then?", and 
"Good, you're being punished after all." 

It got flubbed because I lost my intention to create a shuffle here by commenting it instead of TODOing it. :) I'll adjust this to a properly coded shuffle, thank you!

Reply:
Properly coded in this shuffle that plays after a 'no' answer, resolving. :)